marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/JobControllerTest.java

```diff
@@ -85,4 +87,28 @@ public void testFindStepResponses() throws Exception {
         loginAsTestUserWithRoles("hub-central-user");
         verifyRequestIsForbidden(buildJsonPost(STEP_RESPONSES_PATH, json));
     }
+
+    @Test
+    public void testGetMatchingPropertyValues() throws Exception {
+        String json = "{\n" +
+                "  \"facetName\": \"flowName\",\n" +
+                "  \"searchTerm\": \"simple\"\n" +
+                "}";
+        runAsAdmin();
```

Comment:
You can install a project as a developer, so it's better to use runAsDataHubDeveloper